govcd/vdc_group_test.go

```diff
@@ -0,0 +1,418 @@
+//go:build functional || openapi || vdcGroup || ALL
+// +build functional openapi vdcGroup ALL
+
+/*
+ * Copyright 2021 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+// create only support NSXT VDCs
```

Comment:
what does "support" mean in this context?

Reply:
It means that VDC group can be NSXV or NSXT group. This implementation and tests care only about NSXT ones.

Reply:
do you mean "supported", then?

Reply:
I would say: "supports" as for create function for NSXT VDCs groups

Reply:
```suggestion
// tests creation of  NSX-T VDCs
```

Reply:
changed

Reply:
updated